iib/web/static/api_v1.yaml

```diff
@@ -0,0 +1,458 @@
+          schema:
+            type: boolean
+            example: true
+            default: false
+      responses:
+        '200':
+          description: A list of requests
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  items:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Request'
+                  meta:
+                    $ref: '#/components/schemas/Pagination'
+        '400':
+          description: The query parameters are invalid
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: >-
+                      awesome is not a valid request state. Valid states are:
+                      complete, failed, in_progress
+        '404':
+          description: No requests were found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: The requested resource was not found
+  '/builds/{id}':
+    get:
+      description: Return a specific IIB request
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: The ID of the IIB request to retrieve
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: The requested IIB request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RequestVerbose'
+        '404':
+          description: The request wasn't found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: The requested resource was not found
+    patch:
+      description: Update a request (requires special authorization)
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: The ID of the IIB request to retrieve
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: The request was updated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RequestVerbose'
+        '400':
+          description: The input is invalid
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: The input data must be a JSON object
+        '403':
```

Comment:
Shouldn't this be 401?

Reply:
As we discussed, I added another commit to change the patch request response to `Forbidden` instead of `Unauthorized`. So this stays right :)